client/signup/steps/domains/index.jsx

```diff
@@ -830,9 +830,6 @@ class DomainsStep extends Component {
 			} else if ( 'general-settings' === source && siteSlug ) {
 				backUrl = `/settings/general/${ siteSlug }`;
 				backLabelText = translate( 'Back to General Settings' );
-			} else if ( 'sites-dashboard' === source ) {
-				backUrl = '/sites';
-				backLabelText = translate( 'Back to Sites' );
```

Comment:
Are we good to clean it at this point? It seems it is no longer needed as we use `/sites` as default.

Reply:
> Are we good to clean it at this point?

Where was this introduced? Do all of the relevant flows still work as expected with this removed?

Reply:
I introduced it in https://github.com/Automattic/wp-calypso/commit/aaf547926c3b49463d2d36f20cd6ec61c2b76f62, and at that time, one condition was redirecting to `/sites` and another one to `/sites-dashboard/`. Now both redirects to the same page.

I tested the "add" flow triggered from the sidebar menu, and both the "add" and "import" flows triggered from the Sites page.

Reply:
👍 Sounds good.